config/ext_public-nginx-ingress_publick8s.yaml

```diff
@@ -0,0 +1,7 @@
+controller:
+  replicaCount: 2
+  service:
+    annotations:
+      service.beta.kubernetes.io/azure-load-balancer-internal: false
+    loadBalancerIP: 52.167.253.43
```

Comment:
To be replaced by the correct value

Reply:
- ✅ for the Ipv4 (value should come from a predefined, in terraform, LB)

Reply:
Also, we'll need to add a 2nd Service of type LB (might need a "raw" YAML unless supported by the ingress-nginx chart): https://stackoverflow.com/a/61898643